pkg/traceutil/trace.go

```diff
@@ -181,33 +181,33 @@ func (t *Trace) logInfo(threshold time.Duration) (string, []zap.Field) {
 	var steps []string
```

Comment:
This is no longer needed

Reply:
Hi, Line 207 is still using it~

Reply:
I use tstep here based on t.steps. I mark it as TODO if need.